<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeuronNetworkSettingsDialog</class>
 <widget class="QDialog" name="NeuronNetworkSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конфигурация сети</string>
  </property>
  <layout class="QGridLayout" name="_grid_layout">
   <item row="2" column="0">
    <widget class="QLabel" name="_label_output">
     <property name="text">
      <string>Количество выходов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="_label_layout_count">
     <property name="text">
      <string>Количество слоёв</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="_box_input">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="_box_neutons_in_layout">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="_label_input">
     <property name="text">
      <string>Количество входов</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="_label_neurons_in_layout">
     <property name="text">
      <string>Нейронов в слое</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="_box_layout_count">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="_box_output">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="_label_activation_function">
     <property name="text">
      <string>Функция активации</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="_box_activation_function">
     <item>
      <property name="text">
       <string>Линейная функция</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Однополярный сигмоид</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Биполярный сигмоид</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_button_box</sender>
   <signal>accepted()</signal>
   <receiver>NeuronNetworkSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_button_box</sender>
   <signal>rejected()</signal>
   <receiver>NeuronNetworkSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
